source/_integrations/iotty.markdown

```diff
@@ -8,6 +8,7 @@ ha_iot_class: Cloud Polling
 ha_config_flow: true
 ha_codeowners:
   - '@pburgio'
+  - '@shapournemati-iotty'
```

Comment:
This is not correct, looking upstream in our codebase it says:

![CleanShot 2024-08-02 at 18 18 47@2x](https://github.com/user-attachments/assets/686bc3ca-74c4-488c-b53e-1a034794d410)


Additionally, this field should not be edited manually, it is synced from the codebase.

```suggestion
```

Reply:
I understand. So I'm supposed to make a PR to homeassistant core, right?